django_rq/django_rq

```diff
@@ -1 +0,0 @@
-django_rq
```

Comment:
This file was causing import issues, and I can't work out what it was in the project for. If anyone can shed any light on that will replace?